commands/info.js

```diff
@@ -28,17 +28,24 @@ function displayDB (db) {
   cli.log()
 }
 
-let notDatabaseUrl = (a) => !a.config_vars.find((c) => c === 'DATABASE_URL')
-
 function * run (context, heroku) {
-  let app = context.app
+  const host = require('../lib/host')
+  const fetcher = require('../lib/fetcher')
+  const app = context.app
+  const db = context.args.database
+
+  let addons = []
   let config = heroku.get(`/apps/${app}/config-vars`)
-  let addons = yield heroku.request({
-    method: 'get',
-    path: `/apps/${app}/addons`
-  })
-  addons = addons.filter((a) => a.addon_service.name === 'heroku-postgresql')
-  addons = _.sortBy(addons, notDatabaseUrl, 'name')
+
+  if (db) {
+    addons = yield [fetcher.addon(heroku, app, db)]
```

Comment:
Oh I see, fetcher is new below. So it doesn't. What happens when the addon is not found?